src/controllers/untitledSqlDocumentService.ts

```diff
@@ -0,0 +1,62 @@
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
+
+import VscodeWrapper from './vscodeWrapper';
+import vscode = require('vscode');
+import path = require('path');
+import os = require('os');
+const fs = require('fs');
+
+/**
+ * Service for creating untitled documents for SQL query
+ */
+export default class UntitledSqlDocumentService {
```

Comment:
Fair enough, seems silly they don't have a programmatic way of doing that, but it's a project that's still growing.